src/settings.rs

```diff
@@ -209,19 +209,19 @@ impl Index {
     /// # });
     /// ```
     pub async fn get_settings(&self) -> Result<Settings, Error> {
-        Ok(request::<(), Settings>(
+        request::<(), Settings>(
```

Comment:
Can you give me some idea behind this change? Why it was required before and now isn't?